src/main/java/org/candlepin/subscriptions/metering/service/prometheus/PrometheusMeteringController.java

```diff
@@ -54,20 +61,23 @@
   private final PrometheusService prometheusService;
   private final EventController eventController;
   private final ApplicationClock clock;
-  private final PrometheusMetricsProperties metricProperties;
+  private final PrometheusMetricsProperties promMetricsProperties;
```

Comment:
Nitpick: for consistency:
```suggestion
  private final PrometheusMetricsProperties prometheusMetricsProperties;
```
(similar should be done throughout file).

Reply:
DONE